packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/main/index.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Set up self-service registration
+meta:
+  - name: description
+    content: Okta's self-service registration lets you configure a custom app or the Okta homepage for use when users self-register.
+layout: Guides
+---
+
+This article shows you how to set up Okta's self-service registration (SSR) functionality, which lets you configure a custom app or the Okta homepage to allow users to self-register.
```

Comment:
```suggestion
This article shows you how to set up Okta's self-service registration (SSR) functionality so that you can configure a custom app or the Okta homepage to allow users to self-register.
```